package.json

```diff
@@ -14,6 +14,7 @@
     "@nomiclabs/hardhat-ethers": "^2.0.5",
```

Comment:
We don't need to synchronize the starknet.js version used in the plugin and here?